doc/src/manual/arrays.md

```diff
@@ -1028,33 +1028,57 @@ It is sometimes useful to perform element-by-element binary operations on arrays
 sizes, such as adding a vector to each column of a matrix. An inefficient way to do this would
 be to replicate the vector to the size of the matrix:
 
-```julia-repl
-julia> a = rand(2,1); A = rand(2,3);
+```jldoctest broadcasting
+julia> a = [1, 2]; A = [10 20 30; 40 50 60]
```

Comment:
+1 for not using random numbers. Can these have different letters too? Seems an unnecessary collision, suggest `v` for vector, `M` for matrix?